test/discordBot.test.js

```diff
@@ -3,6 +3,7 @@ const DiscordBot = require('../discordBot')
 require('dotenv').config();
 
 const discordBot = new DiscordBot(process.env.DISCORD_BOT_TOKEN);
+channelId = 'Enter Channel Here';
```

Comment:
we could add another environment for that process.env.SLACK_TEST_CHANNEL_ID

Reply:
yeah that works ill do same for slack since i have something similar there